hl7/4.2/modules/ROOT/pages/index.adoc

```diff
@@ -3,104 +3,55 @@
 
-Release Notes: https://docs.mulesoft.com/release-notes/connector/hl7-connector-release-notes-mule-4[Release Notes] +
-Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-hl7-extension[HL7 EDI Connector]
+For compatibility information and fixed issues, see the HL7 EDI Connector Release Notes.
 
-== Prerequisites
-
-This document assumes that you are familiar with HL7, Mule, Anypoint
-Connectors, Anypoint Studio, Mule flows, and Mule Global Elements.
-
-See the Release Notes for compatibility information. A link to the
-release notes is listed in the See Also section of this document.
-
-You must purchase a license for the HL7 EDI Connector to use it in a production environment.
-
-== POM File Information
-
-[source,xml,linenums]
-----
-<dependency>
-  <groupId>com.mulesoft.connectors</groupId>
-  <artifactId>mule-hl7-extension</artifactId>
-  <version>4.0.3</version>
-  <classifier>mule-plugin</classifier>
-</dependency>
-----
-
-Replace version 4.0.3 for the version you are using. To specify a version, view Anypoint Exchange and click *Dependency Snippets*.
-
-== What's New in this Connector
-
-The data structures used by the HL7 EDI Connector 4.0.0 differ from
-those in the 3.x version used with Mule 3.x.
-
-In the 3.x version of the connector, composite components are effectively in-lined
-at the point of use, so that each individual element
-nested within the composite component is given a key based on the
-containing segment and referenced directly from the map
-representing the segment.
+Standard schema definitions are included in HL7 EDI Connector for
+HL7 versions 2.1, 2.2, 2.3.1, 2.3, 2.4, 2.5, 2.5.1, 2.6, 2.7,
+2.7.1, 2.8, 2.8.1. The schemas are provided in two forms:
 
-In the 4.x version of the connector each
-composite component is instead represented by a child map
-and the values within the composite have keys based on the composite identifier.
+* Standard HL7: using required or optional status data types,
+field lengths, and repetition counts (as listed) for each HL7 version. +
 
-This change in structure allows consistent mappings to be used for
-data in composite components, regardless of the containing
-segment or position within the segment.
+* Lax HL7: all segments and value items are optional. These
+low-level data types are treated as simple strings. Field lengths and repetition counts are not enforced.
 
-== Connect in Design Center
+Both forms of schemas are available in the connector `classpath`. Standard HL7
+schemas use the path: `/hl7/v2_5_1/ADT_A01.esl` +
+Lax schemas use paths: `/hl7lax/v2_5_1/ADT_A01.esl`
 
-The HL7 EDI Connector 4.0 is not supported for use in Design Center.
+The schema definitions you need for xref:hl7-schemas.adoc[HL7 Supplied Schemas] are included.
 
-== Add the Connector to a Studio Project
+For complete information about schemas, see xref:x12-edi-connector::x12-edi-schema-language-reference.adoc[EDI Schema Language Reference].
```

Comment:
Is this link right? Going to X12?

Reply:
Yes, this is a problem that I mentioned that the SLR is really for all these connectors but for now lives in X12 land.